Makefile

```diff
@@ -0,0 +1,7 @@
+.PHONY: vendor
```

Comment:
A beginner's explanation for `.PHONY` can be found [here](https://stackoverflow.com/a/2145605).